lib/classes/Variables.test.js

```diff
@@ -76,6 +74,33 @@ describe('Variables', () => {
         expect(serverless.service.resources.bar).to.equal(barValue);
       });
     });
+
+    it('should delete null environment variables', () => {
+      const serverless = new Serverless();
+
+      serverless.service.provider.environment = {
+        SOME_VAR: 'some-value',
+        OTHER_VAR: null,
+      };
+      serverless.service.functions = {
+        hello: {
+          handler: 'hello.js',
+          environment: {
+            VAR_ONE: 'one',
+            VAR_TWO: null,
+          },
+        },
+      };
+
+      return serverless.variables.populateService().then(() => {
+        /* eslint-disable no-unused-expressions */
```

Comment:
Yes, I agree 👍. disabling that for all tests in the linter config sounds like a good idea (also ran into this a bunch of times).